### DISCUSSION TITLE
In which Blazorise release will ModalService included

### DISCUSSION THREAD

**Discussion by palashjhabak** [Megabit/Blazorise#4018] [Q&A] [answered]
Hi,

I think ModalService-related changes were merged to the main branch but do not seem to be included in the NuGet 1.0.6 release. Is it planned in any of the future releases or am I missing something? 

Thanks,
Palash

PS: Thanks for such a nice Blazor library

**Comment by David-Moreira** [comment, marked as answer]
Hello, it will be on v1.1.0

**Reply by palashjhabak** [reply]
Awesome. thanks for such a quick reply. Is there any way I can use it still?

**Reply by David-Moreira** [reply]
@stsrki can we setup a v1.1.0 preview? Or is it already setup?

**Reply by stsrki** [reply]
It's not yet live but I can make it.
